helix-core/src/main/java/org/apache/helix/monitoring/mbeans/dynamicMBeans/DynamicMBeanProvider.java

```diff
@@ -158,41 +155,39 @@ public synchronized void unregister() {
   }
 
   @Override
-  public Object getAttribute(String attribute)
-      throws AttributeNotFoundException, MBeanException, ReflectionException {
```

Comment:
Please keep these exceptions in the signature. We might want to complete the logic here by throwing the right exceptions moving forward.

Reply:
Right, already added back and implemented.